packages/i18n/src/resBundle.js

```diff
@@ -57,8 +81,19 @@ function clearResBundle () {
 	resBundle = null;
 }
 
+function getIStringFromBundle (str, rb) {
```

Comment:
Can we update `$L.toIString()` instead of creating a semi-duplicate code? I think we can simply add a check for `rb` value and create bundle when not provided.

Reply:
I created this to be used by the new `Text` component to consolidate the key => IString logic. Good catch on the duplication, though! I've refactored `$L` to use this method.